dev/NavigationView/NavigationView.cpp

```diff
@@ -1046,14 +1048,22 @@ void NavigationView::AnimateSelectionChanged(const winrt::IInspectable& prevItem
     }
 }
 
-void NavigationView::PlayIndicatorAnimations(const winrt::UIElement& indicator, float from, float to, bool isOutgoing)
+void NavigationView::PlayIndicatorAnimations(const winrt::UIElement& indicator, float from, float to, winrt::Size beginSize, winrt::Size endSize, bool isOutgoing)
 {
     winrt::Visual visual = winrt::ElementCompositionPreview::GetElementVisual(indicator);
     winrt::Compositor comp = visual.Compositor();
 
     winrt::Size size = indicator.RenderSize();
     float dimension = IsTopNavigationView() ? size.Width : size.Height;
 
+    float beginScale = 1.0f;
+    float endScale = 1.0f;
+    if (IsTopNavigationView())
```

Comment:
>IsTopNavigationView() [](start = 8, length = 21)

To avoid Size.Width == 0, you may need a check like && fabs(size.Width) > 0.001f

Reply:
@licanhua Thanks! I've modified it.